docroot/sites/all/modules/custom/ppgetstat/ppgetstat.module

```diff
@@ -131,12 +197,6 @@ function ppgetstat_cron() {
   // Remove expired cached pages.
   _ppgetstat_cron_remove_cached_pages();
 
-  //Start users sync process with d.org
```

Comment:
@anpolimus, what you can suggest according to removal of these lines? I don't want reinstall the site and wait until history for 12 years will be obtained. These should be run only on production.